test/view_models/you_page_viewmodel_test.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hng/app/app.router.dart';
+import 'package:mockito/mockito.dart';
+import '../helpers/test_helpers.dart';
+import 'package:hng/ui/nav_pages/you_page/you_page_viewmodel.dart';
+
+void main() {
+  group('YouPageViewModelTest -', () {
```

Comment:
also only navigation write test for the user status changing from active to away and back